posthog/plugins/test/plugin_archives.py

```diff
@@ -0,0 +1,4 @@
uc7Rhwwe9j7QY3HKGGBrkYTsYOYYI/xGs9WhxuoQ7BN65GPGh9cznZIdaR+A6uRy0PsbMwq+4bs8eJpLV1z9wA1HtvwdXFzl8ijDbE0TWEkYAbmv7Skob3du9O7s5A61MAgSHoJaAD0pnAybfuQF872Tpf9r0LXQKtI+j9JWIxUHFKMiEf3/wIwWJkiOBQ9+T1U900Q9LPFGi8RxSocu386asTF9jhMg5Iaaed1mNkE+Nv20Sq0PjB972/krXGD60jR+E7YwZb9d7/sZOX230HH1HqTQId4Px51XsrdHXfw97eA0NejLf+x85I9CHi4V3dw9mPE9//Np+Qfy2gUkuz5VqArKDU6lWmIoUZr/A9S2ArzVptDOCE5oXZgVoCL3bwUxZpAuJXqUVVgdJM5mUmBdZkscg2qSxWMMe9QuGfWObSIKhRQIR3KCkqAsuFXqzxyecyk2aXsKU0BWEulQYOJddGLjYZ11BudKkqgfQpwhayWGpkEbkozBOyYg3EKz6gWvMsIyrGN6hekz5YqHKn5WptYK2yVGBxLlAZn2fiRoWmFhmXeQIpz/lKTFsKUTSjsZs62K4FlYiP429hpCrIxkIVRuMzQZfafKxuZSUS4FpWFMhSqzxhFCduqAkE9wpxQ6Go4ctFcITem0p8AEIqeIZYFS2TxffhJ/YXUEsDBAoAAAAIAGNLSVEf4ES4IwAAAC4AAABDAAkAaGVsbG93b3JsZHBsdWdpbi0zYzRjNzdlN2Q3ODc4ZTg3YmUzYzIzNzNiNjU4Yzc0ZWMzMDg1ZjQ5L1JFQURNRS5tZFVUBQABW4+AX1NWyEjNyckvzy/KSSnIKU3PzOMKyC8uychPh0gogGUUoFIAUEsDBAoAAAAIAGNLSVH7pxdtngAAAA0BAABFAAkAaGVsbG93b3JsZHBsdWdpbi0zYzRjNzdlN2Q3ODc4ZTg3YmUzYzIzNzNiNjU4Yzc0ZWMzMDg1ZjQ5L19faW5pdF9fLnB5VVQFAAFbj4BffY3BCsIwDIbvfYrQ0wZjDzDwouy+u4jMLd0KXVOSTn18104EPRgIJH/+749hWiCQxJmmOrh1sl7ALoE4QpfXYy94cr1IBd3ua+/oo1JqSCq0z34JDndz8cOUDQCorWFEA4FpQJErpoBC0JkK8tx8RZdNJlLla71xATlalLOe0TnSFziAfhC7Uf/x3nrOzvSpHsgbO73FD8QYV/Y7q15QSwMECgAAAAgAY0tJUR6z6b+vAAAAHQEAAEUACQBoZWxsb3dvcmxkcGx1Z2luLTNjNGM3N2U3ZDc4NzhlODdiZTNjMjM3M2I2NThjNzRlYzMwODVmNDkvcGx1Z2luLmpzb25VVAUAAVuPgF9tjjEPwiAQhff+iksXF2P3LiYO6ujWmZYrkFCuwhGjTf+7gDF1cGB47/seuaUCqJ2YsG6h1mgtPchbOduojKv3mUZvC2SeQ9s0yrCO/WGgqblR4Cup5v9OYhi8mdmQy/uLR2RgjdBlE4STcCYCASfhP4uB3GhUkpeUUu4T+IafMzsteBfAuPJbko5lXhx+zsUJ7I1TWy9xFNFyRr14bb3HezQeZQKjsAFLv1b5rdUbUEsDBAoAAAAAAGNLSVGjCLPpEgAAABIAAABKAAkAaGVsbG93b3JsZHBsdWdpbi0zYzRjNzdlN2Q3ODc4ZTg3YmUzYzIzNzNiNjU4Yzc0ZWMzMDg1ZjQ5L3JlcXVpcmVtZW50cy50eHRVVAUAAVuPgF9tYXJzaG1hbGxvdz09My44LjBQSwECAAAKAAAAAABjS0lRAAAAAAAAAAAAAAAAOgAJAAAAAAAAABAAAAAAAAAAaGVsbG93b3JsZHBsdWdpbi0zYzRjNzdlN2Q3ODc4ZTg3YmUzYzIzNzNiNjU4Yzc0ZWMzMDg1ZjQ5L1VUBQABW4+AX1BLAQIAAAoAAAAIAGNLSVEcqXWpxwMAAAcHAABEAAkAAAAAAAEAAAAAAGEAAABoZWxsb3dvcmxkcGx1Z2luLTNjNGM3N2U3ZDc4NzhlODdiZTNjMjM3M2I2NThjNzRlYzMwODVmNDkvLmdpdGlnbm9yZVVUBQABW4+AX1BLAQIAAAoAAAAIAGNLSVHBbrBydgIAADAEAABBAAkAAAAAAAEAAAAAAJMEAABoZWxsb3dvcmxkcGx1Z2luLTNjNGM3N2U3ZDc4NzhlODdiZTNjMjM3M2I2NThjNzRlYzMwODVmNDkvTElDRU5TRVVUBQABW4+AX1BLAQIAAAoAAAAIAGNLSVEf4ES4IwAAAC4AAABDAAkAAAAAAAEAAAAAAHEHAABoZWxsb3dvcmxkcGx1Z2luLTNjNGM3N2U3ZDc4NzhlODdiZTNjMjM3M2I2NThjNzRlYzMwODVmNDkvUkVBRE1FLm1kVVQFAAFbj4BfUEsBAgAACgAAAAgAY0tJUfunF22eAAAADQEAAEUACQAAAAAAAQAAAAAA/gcAAGhlbGxvd29ybGRwbHVnaW4tM2M0Yzc3ZTdkNzg3OGU4N2JlM2MyMzczYjY1OGM3NGVjMzA4NWY0OS9fX2luaXRfXy5weVVUBQABW4+AX1BLAQIAAAoAAAAIAGNLSVEes+m/rwAAAB0BAABFAAkAAAAAAAEAAAAAAAgJAABoZWxsb3dvcmxkcGx1Z2luLTNjNGM3N2U3ZDc4NzhlODdiZTNjMjM3M2I2NThjNzRlYzMwODVmNDkvcGx1Z2luLmpzb25VVAUAAVuPgF9QSwECAAAKAAAAAABjS0lRowiz6RIAAAASAAAASgAJAAAAAAABAAAAAAAjCgAAaGVsbG93b3JsZHBsdWdpbi0zYzRjNzdlN2Q3ODc4ZTg3YmUzYzIzNzNiNjU4Yzc0ZWMzMDg1ZjQ5L3JlcXVpcmVtZW50cy50eHRVVAUAAVuPgF9QSwUGAAAAAAcABwBXAwAApgoAACgAM2M0Yzc3ZTdkNzg3OGU4N2JlM2MyMzczYjY1OGM3NGVjMzA4NWY0OQ==",
```

Comment:
What's this?

Reply:
That's a base64 zip file used for testing installing the plugin without needing to do a real request to github. I added a quick comment to the file about it.